data/human/outfits.txt

```diff
@@ -391,6 +391,16 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "Combat Methamphetamines"
+	plural "Combat Methamphetamines"
+	category "Hand to Hand"
+	cost 10000
+	thumbnail "outfit/drugs"
+	"capture attack"  2.2
+	"capture defense" 1.3
+	"illegal" 800000
+	"unplunderable" 1
+	description "Although they decrease your crew's judgement and are illegal, these narcotics can increase your crew's fighting spirit and will numb a certain amount of pain."
```

Comment:
What does "decrease your crew's judgement" mean? Do they lower your crew's critical thinking skills, or lower their opinion of First Last, or both?

Also, 2.2 attack/1.3 defense is pretty powerful - almost as offensive as Nerve Gas, and almost as defensive as a Protection Station, without requiring any extra outfit space. This item needs some major consequences to not be crazy OP. My first thought would be to add a mission chain which starts out like Bad Trip and ends with First Last overdosing, but that is probably too dark for this game.

It would be great if we could programmatically remove crew with events, but I don't think we can - I feel a sufficient consequence would be a crewmember dying after launch, on a smaller ship possibly majorly impacting their ability to maneuver, and hopefully forcing the player to reconsider their decision to drug up their crew.

Reply:
"What does "decrease your crew's judgement" mean? Do they lower your crew's critical thinking skills, or lower their opinion of First Last, or both?"

Never saw "judgement" used for reputation/opinion, so I assume it means reasoning/thinking skills.

Reply:
@AbsolutelyLudicrous I had tried to figure out how code missions with crew members, but from what I can see there is no part of the code that will allow me to do it.

I should probably change judgement to perception.